db/migrate/20230224185802_create_ingredients.rb

```diff
@@ -0,0 +1,20 @@
+class CreateIngredients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ingredients do |t|
+      t.string :name
+      t.string :type
+      t.boolean :is_alcoholic
+      t.date :vintage
+      t.string :size
+      t.float :abv
+      t.string :purchase_url
+      t.blob :image
+      t.json :metadata
+
+      t.timestamps
```

Comment:
```suggestion
    create_table :ingredients do |t|
      t.timestamps
      t.string :name
      t.string :type
      t.boolean :is_alcoholic
      t.date :vintage
      t.string :size
      t.float :abv
      t.string :purchase_url
      t.blob :image
      t.json :metadata
```
This is just an aesthetic choice, but I would recommend putting the timestamps first for all tables so they show up in the db as the first two columns after `id`.